config.schema.json

```diff
@@ -55,9 +55,10 @@
                 "title": "Show docking status as a contact sensor",
                 "required": false
             },
-            "noDockOnStop": {
+            "DockOnStop": {
```

Comment:
I'd like to follow the same style as the other config properties and start with a lowercase d